packages/web/src/apollo/index.tsx

```diff
@@ -18,6 +18,7 @@ const {
   useQuery,
   useMutation,
   useSubscription,
+  useSuspenseQuery,
```

Comment:
How does this know to use the 3.8.1 client vs the exp next js?

Reply:
This one doesn't :)

As part of setting up ssr, you will need to change the apollo provider in your App.tsx


```diff
- import { RedwoodApolloProvider } from '@redwoodjs/web/apollo'
+ import { RedwoodApolloProvider } from '@redwoodjs/web/dist/apollo/suspense'
```